### PR TITLE
Add custom cursor release notes

### LINKED ISSUE #1670: Write release notes for PR #14284: Add custom cursors
https://github.com/bevyengine/bevy/pull/14284 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `/home/alice/Documents/bevy-website/release-content/0.15/release-notes/14284_Add_custom_cursors.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by eero-lehtinen**
Fixes #1670

I have a video too but not sure where to put it. 

https://github.com/user-attachments/assets/90dd5ca1-c423-4a83-a303-042423524db5

### CODE REVIEW

**Comment by mgi388** [comment]
@eero-lehtinen ‘custom cursor images’ also mean ‘custom animated cursors’ are now possible aren’t they? I think your video is showing that but it’s not 100% obvious to me. 

I think the answer is: yes, but it could be worth clarifying in the notes.

**Comment by mgi388** [comment]
Btw, by animations I don’t mean “native” I just mean presumably one could  use a texture atlas and change the texture atlas index every, say, 100ms to build their own animated cursors. Feel free to tell me I’m off topic, I just personally want animated custom cursors so was curious if you knew
If it was supported because mentioning in release notes would make this an even better feature :)

**Comment by eero-lehtinen** [comment]
Yeah animated cursors are very much possible by changing the image every frame.

File: release-content/0.15/release-notes/14284_Add_custom_cursors.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <!-- Add custom cursors -->
 <!-- https://github.com/bevyengine/bevy/pull/14284 -->
 
-<!-- TODO -->
+Previously Bevy's native window cursors supported only a fixed set of built-in OS cursors. Now they also support arbitrary images as "custom cursors". Custom cursors still use native facilities of the OS, which allows them to stay perfectly responsive even when the frame rate of the application drops.
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/14284_Add_custom_cursors.md:4]: We should demonstrate how to use this feature, or at the very least link to docs about how to do so.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**